### ISSUE TITLE
Can't run on Virtual Machine

### ISSUE THREAD

**Issue by connebeest** [CryptoDredge/miner#5]
I'm sure this is intended, but wanted to check why this is a limitation?

![image](https://user-images.githubusercontent.com/10919552/192740830-f82cd89d-a470-4875-90ed-bbd3a1fc50c8.png)

Thanks for clarifying. I personally have the need to be able to run this in a virtual machine, so if this limitation could be lifted, that would be great.

**Comment by saymyname77** [comment]
You're absolutely right, it's almost impossible to use on a cloud host, which is very inconvenient.
